crypto/castsb.h

```diff
@@ -0,0 +1,554 @@
+/****************************************************************************
+ * crypto/castsb.h
+ * $OpenBSD: castsb.h,v 1.1 2000/02/28 23:13:04 deraadt Exp $
+ *
+ * CAST-128 in C
+ * Written by Steve Reid <sreid@sea-to-sky.net>
+ * 100% Public Domain - no warranty
+ * Released 1997.10.11
+ ****************************************************************************/
+
+static const u_int32_t cast_sbox1[256] =
```

Comment:
Do we need to have this available to public? Maybe better move arrays to C file and provide `extern` reference via header?

Reply:
this buffer just use castsb,not need `extern`

Reply:
I'm just thinking that each file that includes this header will have a own copy of array.

Reply:
yes,but he is only included once by cast.c